src/components/tooltip/tooltip.scss

```diff
@@ -39,32 +39,32 @@ md-tooltip {
       padding-left: $tooltip-lr-padding-lg;
       padding-right: $tooltip-lr-padding-lg;
     }
-    &.md-show-add {
+    &._md-show-add {
       transition: $swift-ease-out;
       transition-duration: .2s;
       transform: scale(0);
       opacity: 0;
     }
-    &.md-show, &.md-show-add-active {
+    &._md-show, &._md-show-add-active {
       transform: scale(1);
       opacity: 1;
       transform-origin: center top;
     }
-    &.md-show-remove {
+    &._md-show-remove {
       transition: $swift-ease-out;
       transition-duration: .2s;
-      &.md-show-remove-active {
+      &._md-show-remove-active {
         transform: scale(0);
         opacity: 0;
       }
     }
   }
 
-  &.md-hide {
+  &._md-hide {
```

Comment:
@EladBezalel Similar thing here, I would expect a JS change to go along with this, but see no references. Does this have to do with the `ngShow`/`ngHide` directives? If so, I did not make any similar changes.